src/inactive-regions.ts

```diff
@@ -0,0 +1,94 @@
+import * as vscode from 'vscode';
+import * as vscodelc from 'vscode-languageclient/node';
+
+import {ClangdContext} from './clangd-context';
+
+// Parameters for the inactive regions (server-side) push notification.
+// Mirrors the structure in the inactive regions proposal for LSP.
```

Comment:
is there a related GitHub thread for this? If so, it would be nice to mention it (I tried to search `inactive regions proposal for LSP`, it didn't find anything).

Reply:
The comment dates back to [this 2019 patch](https://reviews.llvm.org/D67537?vs=on&id=220043#toc). Unfortunately, I no longer remember what it was referring to :/

I do see a similar structure used in both [cquery](https://github.com/jacobdufault/cquery/blob/9b80917cbf7d26b78ec62b409442ecf96f72daf9/src/message_handler.cc#L15-L22) and [vscode-cpptools](https://github.com/microsoft/vscode-cpptools/blob/c417da6ed438327a39b7fa0d18bfd4d05e3279cc/Extension/src/LanguageServer/client.ts#L222), but I can't find an actual LSP proposal based on either of these.

I will remove this comment line to avoid confusion. (And we can make an LSP proposal ourselves once clangd has this support.)